pkgs/development/libraries/medfile/default.nix

```diff
@@ -9,6 +9,10 @@ stdenv.mkDerivation rec {
     sha256 = "1khzclkrd1yn9mz3g14ndgpsbj8j50v8dsjarcj6kkn9zgbbazc4";
   };
 
+  patches = [
+    ./hdf5-1.12.patch
```

Comment:
Could this patch be fetched instead?

Reply:
I built it from scratch so unfortunately no.  Where and how the development of medfile/medfichier happens is a bit unclear how we can work with upstream on this :disappointed:

Reply:
Well thats unfortunate.